### PR TITLE
dont show autotranslate for organisation, fixes #318

### LINKED ISSUE #318: Automatic Translations w/o publish_pages
Without the right "publish_pages" and the activated revisionary plugin, the Automatic Translation button will not work as intended. ~~Most likely the translations will not be saved at all.~~ Translations will be published without review / status of source page.

### PR DESCRIPTION
**Pull request by sven15**
Organisation should not be able to use autotranslate button -> publish_pages required

### CODE REVIEW

**Comment by Meash** [comment]
Why does an organization not have the right to translate their own content?
They shouldn't be able to publish it but they should be able to create the translation.

**Comment by sven15** [comment]
Yes, it would be better, but the this requires taking over the status of the original page to the translated page. This is a quick fix.

**Comment by Meash** [comment]
But we already copy the `post_status`: https://github.com/Integreat/cms/blob/2efbed4a2b9620c56bf1499e8ae574cd3f65f196/wp-content/plugins/automatic-translation/TranslationService.php#L22
Can you elaborate what exactly is the issue here?

**Comment by sven15** [comment]
For example non-Verwalter can overwrite approved (manual) translations with automatic translations. Maybe the Verwalter / Vertrauenswürdige Organisation indended to keep the manual translation because the changes in german are small.

**Comment by Meash** [comment]
Okay I see, that makes sense then. Is there an issue for what you described? Just so we don't forget about it..

File: wp-content/plugins/automatic-translation-button/plugin.php [hunk 1]
```diff
@@ -12,6 +12,12 @@
  */
 class AutomaticTranslationButtonPlugin {
 	public function enqueue_styles_and_scripts() {
+		
+		//dont show autotranslate button if user is organisation
+		if( ! current_user_can( 'publish_pages' ) ) {
+			return false;
```
Comment (Meash) [inline wp-content/plugins/automatic-translation-button/plugin.php]: I'm not sure if the `false` doesn't stop further enqueue propagation. Does anything stop us from `return;` without the `false`?

**Comment by sven15** [comment]
Issue created.

**Comment by sven15** [comment]
works in linux. i think that is enough for merging, as all our servers are in linux.

**Comment by Meash** [comment]
see #354

**Pull request merged**